src/api/utils/responseUtils.ts

```diff
@@ -0,0 +1,42 @@
+import { Response } from "express";
+import { ApiResult } from "../interfaces/ApiResult";
+
+/**
+ * Handles sending of result for successful API calls.
+ *
+ * @param res response to call
+ * @param statusCode status code to return
+ * @param data data to send in result
+ * @param message message to send in result
+ */
+function sendSuccessResponse(res: Response, statusCode: number, data: object | unknown[] | null, message: string) {
```

Comment:
Good point! I separated it partly just for the code to be a little more easy to read because we can tell from `sendSuccessResponse` that this is a success case and `sendErrorResponse` that this is a failed case without even eyeing the parameters. Happy to have discussions to change this!